e2e/model/sig-triplet.go

```diff
@@ -0,0 +1,6 @@
+package model
```

Comment:
- The name of the file suggests it is a `triplet` but the name of the struct is `duplet`. 
- I think that `duplet` is actually `pair`